src/main/java/nl/knaw/dans/dvcli/command/DatasetValidateFiles.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2024 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.dvcli.command;
+
+import nl.knaw.dans.dvcli.action.ConsoleReport;
+import nl.knaw.dans.dvcli.action.Pair;
+import nl.knaw.dans.dvcli.action.SingleIdOrIdsFile;
+import nl.knaw.dans.dvcli.action.ThrowingFunction;
+import nl.knaw.dans.lib.dataverse.AdminApi;
+import nl.knaw.dans.lib.dataverse.DataverseException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+
+import java.io.IOException;
+import java.util.List;
+
+@Command(name = "validate-files",
+         mixinStandardHelpOptions = true,
+         description = " Make sure that all files have correctly stored in object storage.")
+public class DatasetValidateFiles extends AbstractCmd {
+    @ParentCommand
+    private DatasetCmd datasetCmd;
+
+    protected List<Pair<String, IdParam>> getIds(DatasetCmd datasetCmd) throws IOException {
+        List<String> pids = new SingleIdOrIdsFile(datasetCmd.targets, SingleIdOrIdsFile.DEFAULT_TARGET_PLACEHOLDER).getPids().toList();
+        if (!pids.isEmpty())
```

Comment:
I think you can drop this line and the second return.

Reply:
Done.